modules/oaiharvest/lib/oai_harvest_daemon.py

```diff
@@ -572,36 +572,47 @@ def remove_duplicates(harvested_file_list):
     """
     Go through a list of harvested files and remove any duplicate records.
     """
+    from copy import deepcopy
+    import lxml.etree as etree
```

Comment:
We can require lxml, but better only for master, as I mentioned in https://github.com/inveniosoftware/invenio/pull/2601#discussion_r22522187 .  Do you need this fixed for maint-1.1?  If yes, then throwing a warning "please consider installing lxml" may be acceptable, if we can mostly keep the old behaviour for those maint-1.1 that are not affected by this due to the OAI sources they use...